models/fetchArticles.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection.js");
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT articles.article_id, articles.author, articles.title, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comment_id) AS comment_count
+      FROM articles
+      JOIN comments ON comments.article_id = articles.article_id
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at DESC;`
```

Comment:
Lovely SQL query, nicely formatted, you selected only the columns you want and great to see you've used the `COUNT` aggregate query, well done!